packages/wp-now/src/download.ts

```diff
@@ -194,12 +202,32 @@ export async function downloadWordPress(
 }
 
 export async function downloadSqliteIntegrationPlugin() {
-	return downloadFileAndUnzip({
+	// Remove the old SQLite plugin if it exists
+	const oldSqlitePath = path.join(getWpNowPath(), `${SQLITE_FILENAME}-main`);
+	if (fs.existsSync(oldSqlitePath)) {
+		fs.rmSync(oldSqlitePath, {
+			recursive: true,
+		});
+	}
+
+	await downloadFileAndUnzip({
 		url: SQLITE_URL,
-		destinationFolder: getWpNowPath(),
+		destinationFolder: path.join(getWpNowPath(), 'mu-plugins'),
 		checkFinalPath: getSqlitePath(),
 		itemName: 'SQLite',
 	});
+
+	// Replace {SQLITE_IMPLEMENTATION_FOLDER_PATH} with the actual path
+	fs.writeFileSync(
+		getSqliteDbCopyPath(),
+		fs
+			.readFileSync(getSqliteDbCopyPath())
+			.toString()
+			.replace(
+				"'{SQLITE_IMPLEMENTATION_FOLDER_PATH}'",
+				`realpath( __DIR__ . '/mu-plugins/${SQLITE_FILENAME}-main' )`
```

Comment:
Why use the `-main` suffix?

Reply:
Because this is the plugin name inside mu-plugins.